app/controllers/users_controller.rb

```diff
@@ -14,8 +14,25 @@ def create
     end
   end
 
+  def edit
+    @user = User.find(current_user.id)
+  end
+
+  def update
+    @user = User.find(current_user.id)
+    if @user.update(user_update)
```

Comment:
Let's consider a use case:
- Current user's id is 2. So your edit URL should be `users/2/edit`.
- Now the use user manually hits a url in the browser i.e `users/1/edit`

What will happen?